content/docs/start/experiments.md

```diff
@@ -21,39 +21,34 @@ the [`example-dvc-experiments`][ede] project.
 
 <details>
 
-### ⚙️ Installing the example project
+### ⚙️ Initializing a project with DVC experiments
 
-These commands are run in the [`example-dvc-experiments`][ede] project. You can
-run the commands in this document after cloning the repository, installing the
-requirements, and pulling the data.
+If you already have a DVC project, that's great. You can start to use `dvc exp`
+commands right away to run experiments in your project. (See the [user's guide]
+for detailed information.) Here, we briefly discuss how to structure an ML
+project with DVC experiments using `dvc exp init`.
 
-#### Clone the project and create virtual environment
+[user's guide]: /doc/user-guide/experiment-management/
 
-Please clone the project and create a virtual environment.
-
-> We strongly recommend to create a virtual environment to keep the libraries we
-> use isolated from the rest of your system. This prevents version conflicts.
+A typical machine learning project has data, a set of scripts that train a
+model, a bunch of hyperparameters that tune training and models, and outputs
+metrics and plots to evaluate the models. DVC makes certain assumptions about
+the names of these elements to initialize a project with:
 
 ```dvc
-$ git clone https://github.com/iterative/example-dvc-experiments -b get-started
-$ cd example-dvc-experiments
-$ virtualenv .venv
-$ . .venv/bin/activate
-$ python -m pip install -r requirements.txt
+$ dvc exp init python src/train.py
 ```
 
-#### Get the data set
+Here, `python src/train.py` specifies how you run experiments. It could be any
+other command.
 
-The repository we cloned doesn't contain the dataset. Instead of storing the
-data in the Git repository, we use DVC to retrieve from a shared data store. In
-this case, we use `dvc pull` to update the missing data files.
+If your project uses different names for them, you can set directories for
+source code (default: `src`), data (`data/`), models (`models/`), plots
+(`plots/`), and files for hyperparameters (`params.yaml`), metrics
+(`metrics.json`) with the options supplied to `dvc exp init`.
 
-```dvc
-$ dvc pull
-```
-
-The repository already contains the necessary configuration to run the
-experiments.
+You can also set these options in a dialog format with
+`dvc exp init --interactive`.
```

Comment:
I think `exp init -i` is the recommended way for beginners (as mentioned in https://dvc.org/doc/user-guide/experiment-management/experiments-overview#initialize-dvc-experiments-on-any-project).

You may want to double check with @dberenbaum

Reply:
We go back and forth on this. I think it's fine here if the goal is to keep it succinct and get something merged. I believe @iesahin and I have discussed before, so I assume this was a conscious choice.